Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
```

Comment:
Testing a reply.

---
In reply to: [58287580](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r58287580) [](ancestors = 58287580)